text/0079-create-stackify-repo.md

```diff
@@ -3,6 +3,7 @@
 - Name: Create stackify repo
 - Start Date: 2020-11-06
 - Authors: @martyspiewak @mdelillo @dumez-k
+- Status: Approved
```

Comment:
```suggestion
- Status: Superseded
```

I am assuming this is superseded by [RFC 0096](https://github.com/buildpacks/rfcs/blob/main/text/0096-remove-stacks-mixins.md)?

cc @sclevine

Reply:
I apologize for missing this.

_Addressed in latest commit._